lib/medtronic600/medtronic600Simulator.js

```diff
@@ -493,6 +359,12 @@ class Medtronic600Simulator {
             return false;
           }
         }
+      } else if (event.type === 'basal') {
+        // Filter out any basals with no duration. This can happen inside
+        // applySuspendedBasals(), when a basal is suppressed completely by a pump suspend
```

Comment:
Don't you want to know when a basal is being suppressed? This sounds like useful info that we may not have had for other pumps, i.e., maybe you don't even have to look up basal schedules if basals are still coming through during a suspend. Or am I misunderstanding what's happening here?